CMakeLists.txt

```diff
@@ -30,7 +30,7 @@ cmake_minimum_required(VERSION 3.28)
 # outdated registry entries.
 # ref: https://cmake.org/cmake/help/latest/variable/CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION.html
 if(CMAKE_GENERATOR MATCHES "Visual Studio")
-    set(CMAKE_SYSTEM_VERSION 10.0 CACHE INTERNAL "target Windows SDK version" FORCE)
+  set(CMAKE_SYSTEM_VERSION 10.0 CACHE INTERNAL "target Windows SDK version" FORCE)
```

Comment:
What's happening with all of these whitespace changes in `CMakeLists.txt`? Are they tabs-to-spaces changes being made by your editor or something?

If so, please revert them.

Reply:
Sorry, I keep hitting standard `save` by accident (which autoformats) 😅